torchaudio/functional.py

```diff
@@ -245,15 +245,15 @@ def create_dct(n_mfcc, n_mels, norm):
     outdim = n_mfcc
     dim = n_mels
     # http://en.wikipedia.org/wiki/Discrete_cosine_transform#DCT-II
-    n = np.arange(dim)
-    k = np.arange(outdim)[:, np.newaxis]
-    dct = np.cos(np.pi / dim * (n + 0.5) * k)
+    n = torch.arange(dim, dtype=torch.float32)
```

Comment:
Don't pass the type to the constructor in case someone wants to change the default dtype from float32 to float64 (for e.g. debugging purposes).